lib/stripe/api_operations/request.rb

```diff
@@ -30,6 +32,18 @@ def request(method, url, params={}, opts={})
 
           [resp, opts_to_persist]
         end
+
+        private
+
+        KNOWN_OPTS = Set[:api_key, :idempotency_key, :stripe_account, :stripe_version]
```

Comment:
I don't think the `private` block will affect visibility for constants, so can put in the `private_constant` tag like we talked about?